packages/react-sdk/src/components/BoardBar/pdf/pdf.local.ts

```diff
@@ -14,18 +14,17 @@
  * limitations under the License.
  */
 
-// Always make sure to initialize fonts if PDFs are generated.
-import './initializeFonts';
-
 import * as pdfMake from 'pdfmake/build/pdfmake';
 import { TDocumentDefinitions } from 'pdfmake/interfaces';
+import { initializeFonts } from './initializeFonts';
 
 /**
  * Generate the PDF in a separate module that can either:
  *   1. be imported by a web worker
  *   2. be imported lazily as a separate module
  */
 export function generatePdf(content: TDocumentDefinitions): Promise<Blob> {
-  const pdf = pdfMake.createPdf(content);
+  const cfg = initializeFonts();
```

Comment:
I wonder if the name still make sense.
What about to rename it to `createFontConfig` ?

Reply:
Sure, makes sense.